netlify.toml

```diff
@@ -10,7 +10,7 @@
   package = "@netlify/plugin-sitemap"
 
   [plugins.inputs]
-
+    filePath = "docs/sitemap.xml"
```

Comment:
Please move filePath below the buildDir line, for reference see  [filePath = "docs/sitemap.xml"
    buildDir = "public/"](https://github.com/netlify-labs/netlify-plugin-sitemap#writing-to-a-subfolder-path)
    
```
buildDir = "public/"
filePath = "docs/sitemap.xml"
```